src/Codeception/Module/FTP.php

```diff
@@ -836,15 +837,30 @@ private function modified($filename)
      */
     protected function sftpConnect($user, $password)
     {
-        $this->ftp = new \Net_SFTP($this->config['host'], $this->config['port'], $this->config['timeout']);
+        if (class_exists('Net_SFTP')) {
+            $this->ftp = new \Net_SFTP($this->config['host'], $this->config['port'], $this->config['timeout']);
+        } elseif (class_exists('phpseclib\Net\SFTP')) {
+            $this->ftp = new \phpseclib\Net\SFTP($this->config['host'], $this->config['port'], $this->config['timeout']);
+        } else {
+            throw new ModuleException('phpseclib/phpseclib library is not installed');
+        }
+
         if ($this->ftp === false) {
             $this->ftp = null;
             $this->fail('failed to connect to ftp server');
         }
 
         if (isset($this->config['key'])) {
             $keyFile = file_get_contents($this->config['key']);
-            $password = new \Crypt_RSA();
+
+            if (class_exists('Crypt_RSA')) {
+                $password = new \Crypt_RSA();
+            } elseif (class_exists('phpseclib\Net\SFTP')) {
+                $password = new \phpseclib\Crypt\RSA();
```

Comment:
I apologise for nitpicking, but checking one class and instantiating another doesn't look right to me.

Reply:
you're right, bad copy/past :p fixed